CHANGELOG.md

```diff
@@ -26,6 +26,25 @@ release.
   clients invoking them.
 - BREAKING: Rename all JVM metrics from `process.runtime.jvm.*` to `jvm.*`
   ([#241](https://github.com/open-telemetry/semantic-conventions/pull/241))
+- BREAKING: Add namespaces to JVM metric attributes ([#20](https://github.com/open-telemetry/semantic-conventions/pull/20)).
+  - Rename attributes `type` to `jvm.memory.type`, `pool` to `jvm.memory.pool.name`
+  - Applies to metrics:
+    - `process.runtime.jvm.memory.usage`
+    - `process.runtime.jvm.memory.init`
+    - `process.runtime.jvm.memory.committed`
+    - `process.runtime.jvm.memory.limit`
+    - `process.runtime.jvm.memory.usage_after_last_gc`
```

Comment:
For the purposes of the changelog, do we want to reference the old metric names with the `process.runtime.*` prefix or the new shorter ones?

Reply:
makes sense to me to use the names that were in effect with this change (so the new shorter ones), I've made this change